### ISSUE TITLE
Detect if the number is prime

### ISSUE THREAD

**Issue by r3tec** [r3tec/cs#9]
As a developer I need a function that given a number returns true if it is prime, false otherwise
- [ ] Consult [readme.md](https://github.com/r3tec/cs/blob/master/README.md)

**Comment by AshutoshMalviya** [comment]
in javascript 
function isPrime(num) {
    for(let i=2;i<=num/2;i++){
        if(num%i==0) {
            return false;
        } 
    }
    return true;
}

**Comment by AshutoshMalviya** [comment]
it seems like you are learning  github. I am on the same page but doing things in private repository

**Comment by r3tec** [comment]
No. I am running a computer school. Your solution is extremely inefficient

**Comment by AshutoshMalviya** [comment]
Sorry sir, just a beginner here. i will retry to come with an efficient solution.

**Comment by r3tec** [comment]
@AshutoshMalviya It is not a trivial exercise, btw. I plan to have several sessions to explore it. My school is free to join, if you put your email on your profile I can send you some info.

**Comment by YashThakare** [comment]
In Javascript: 
//Sieve of Eratosthenes algorithm
var eratosthenes = function(n) {
    var array = [], upperLimit = Math.sqrt(n), output = [];
    for (var i = 0; i < n; i++) {
        array.push(true);
    }

    // Remove multiples of prime no
    for (var i = 2; i <= upperLimit; i++) {
        if (array[i]) {
            for (var j = i * i; j < n; j += i) {
                array[j] = false;
            }
        }
    }

    // All array[i] set to true are primes
    for (var i = 2; i < n; i++) {
        if(array[i]) {
            output.push(i);
        }
    }

    return output;
};

//Time Complexity:O(O(sqrt(n)loglog(n))

**Comment by YashThakare** [comment]
In Javascript:
//To check whether a number is prime or not
function isPrime(num) {
  var sqrt_num=Math.floor(Math.sqrt(num));
    var prime = num != 1;
    for(var i=2; i<sqrt_num+1; i++) { // sqrt_num+1
        if(num % i == 0) {
            prime = false;
            break;
        }
    }
    return prime;
}

**Comment by r3tec** [comment]
@YashThakare Thank you, but this is not a Stackoverflow question. If you want to contribute please follow readme.md

**Comment by r3tec** [comment]
@YashThakare This project is a playground for my computer school. We will review multiple solutions when they appear in tested code in pull requests. Please check out readme.md
